ui/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/RemoteEnvironment.scala

```diff
@@ -163,7 +162,7 @@ trait StandardRemoteEnvironment extends FailFastCirceSupport with RemoteEnvironm
     EitherT {
       invokeStatus(HttpMethods.GET, List("processes", localProcess.id)).flatMap { status =>
         if (status == StatusCodes.NotFound)
-          invokeForSuccess(HttpMethods.POST, List("processes", localProcess.id, category))
+          invokeForSuccess(HttpMethods.POST, pathParts = List("processes", localProcess.id, category), queryString = Some(s"isSubprocess=${localProcess.metaData.isSubprocess}"))
```

Comment:
can we do it in more "typesafe" way? not glueing strings like that?

Reply:
Done. For no query string, single `?` character is not appended anymore. I extracted some methods that invoke API - please have a look at last commit.